package/MDAnalysis/lib/_cutil.pyx

```diff
@@ -90,6 +91,47 @@ def unique_int_1d(np.intp_t[:] values):
     return np.array(result)
 
 
+@cython.boundscheck(False)
+def _in2d(np.intp_t[:, :] arr1, np.intp_t[:, :] arr2):
```

Comment:
We probably should have a direct test for this rather than just indirectly testing via hbond_autocorrel (it's a useful function that probably would have other uses).